src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs

```diff
@@ -325,6 +374,9 @@ public Task<IEnumerable<NotificationHubDescription>> GetNotificationHubsAsync()
             throw new NotImplementedException();
         }
 
+        /// <summary>
```

Comment:
If we are going to add XML comments, add the complete XML parts including the parameter